webcompat/webhooks/model.py

```diff
@@ -274,14 +293,21 @@ def process_issue_action(self):
             else:
                 return make_response('outreach generator url added', 200)
         elif self.action == 'opened' and scope == 'private':
-            # webcompat-bot needs to comment on this issue with the URL
+            # webcompat-bot needs to comment public URL of the issue
+            # and we try to classify the issue using bugbug
             try:
                 self.comment_public_uri()
             except HTTPError as e:
                 msg_log(f'comment failed ({e})', self.number)
                 return oops()
-            else:
-                return make_response('public url added', 200)
+
+            try:
+                self.classify()
+            except HTTPError as e:
+                msg_log(f'classification failed ({e})', self.number)
```

Comment:
let's test the error message.

Reply:
Added a test for that in test_webhook_model.py

Reply:
Yes perfect.